BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/FileDiagnosticsModule/TraceSourceForEventSource.cs

```diff
@@ -13,7 +13,12 @@
     internal class TraceSourceForEventSource : TraceSource, IEventListener, IDisposable
     {
         private const long AllKeyword = -1;
+
+#if REDFIELD
```

Comment:
Do we need to change this?

Reply:
This might not be needed, but when we capture data using `FileDiagnostics` we will get a source name with `RedField` prefixed. We might not rely on this feature anymore, we should use `SelfDiagnostics` feature going forward.